bindings/python/multibody/data.hpp

```diff
@@ -115,11 +116,9 @@ namespace pinocchio
                          "It contains all the data that can be modified by the algorithms.",
                          bp::no_init)
         .def(DataPythonVisitor());
-        
-        bp::class_< container::aligned_vector<Vector3> >("StdVec_vec3d")
-        .def(bp::vector_indexing_suite< container::aligned_vector<Vector3>, true >());
-        bp::class_< std::vector<int> >("StdVec_int")
-        .def(bp::vector_indexing_suite< std::vector<int> >());
+        StdAlignedVectorPythonVisitor<Vector3, true>::expose("StdVec_vec3d");
+        StdAlignedVectorPythonVisitor<Matrix6x, true>::expose("StdMat_Matrix6x");
```

Comment:
I might be late, but shouldn't this be `StdVec_Matrix6x` instead of `StdMat_Matrix6x`?

Reply:
yes, it should be. i'll fix this

Reply:
I created a PR for this.